### ISSUE TITLE
Framhäv bra svenska plugins/teman

### ISSUE THREAD

**Issue by khromov** [wpsvse/wpsv.se#22]
Artikelserie och/eller en fristående sida med bra tillägg/teman från svenska WP-utvecklare.

**Comment by marcustisater** [comment]
kan nog vara schyst att kombinera båda, vi börjar med en fristående sida.

**Comment by marcustisater** [comment]
hur är tanken, en CPT med fält som wporgnamn, projektets namn, projektets url, description? och sen så kör vi en listning på det rakt upp och ner?

Om vi pratar om 50 tillägg så kanske man kan ha en checkbox där det står "framhävd" så kan man försöka lyfta upp de lite snyggare med stilsättning.

Får jag försöka på mig en skiss? :)

**Comment by khromov** [comment]
@marcustisater Ja, det låter bra. Sen tror jag vi skriver ett vanligt blogginlägg och belyser ett par st. varje månad och lägger in i "databasen". Skissa gärna. :)

**Comment by tobifjellner** [comment]
Jag kan se flera infallsvinklar på detta.
För att puffa för den svenska communityn och få medmänniskor att vara stolta över vad vi kan är det jätteul med personporträtt, intervjuer och beskrivningar av projekt där svenskar är inblandade. Och det kan gälla allt från superkomplicerade tillägg som gör vardagen ljusare för några andra superhjärnor runtom i världen, till temagranskning, vikten av felrapporter, responsible disclosure o.s.v.
Men för en "vanlig wp-användare" finns det också ett värde i att tipsa om bra, välbyggda tillägg och teman, oavsett upphovsman. Där kan vi också puffa för teman och tillägg som nyligen blivit tillgängliga på svenska (och samtidigt förklara lite hur hela processen hänger samman).
